AutoCollection/ServerRequests.ts

```diff
@@ -151,7 +151,7 @@ class AutoCollectServerRequests {
         var requestParser = _requestParser || new ServerRequestParser(request, correlationContext && correlationContext.operation.parentId || Util.newGuid());
 
         if (Util.canIncludeCorrelationHeader(client, requestParser.getUrl())) {
```

Comment:
I'm not sure, @KamilSzostak added in this check and may have more context. Perhaps it is because `trackRequestSync` is never called internally, so it must be called by the end-user and thus we want to track the call ignoring blacklists, but `trackRequest` here is also used internally?